nss-day-cohort-46/rare-client-groovy-goblins
# Cr categories create

# Description

User can create a new category.
Categories are listed alphabetically.

## Type of change


# Testing Instructions

`git fetch --all`
`git checkout cr-categories-create`
`npm start`

For server-side instructions, see backend pull request.
After success login, click `Category Management` and click on `Create Category`.
After creating  and saving a new category, user will see list of categories alphabetically. 

# Checklist:
